lib/jekyll-seo-tag/drop.rb

```diff
@@ -0,0 +1,239 @@
+module Jekyll
+  class SeoTag
+    class Drop < Jekyll::Drops::Drop
+      include Jekyll::SeoTag::JSONLD
+
+      TITLE_SEPARATOR = " | ".freeze
+      FORMAT_STRING_METHODS = %i[
+        markdownify strip_html normalize_whitespace escape_once
+      ].freeze
+
+      def initialize(text, context)
+        @obj = {}
+        @mutations = {}
+        @text    = text
+        @context = context
+      end
+
+      def version
+        Jekyll::SeoTag::VERSION
+      end
+
+      # Should the `<title>` tag be generated for this page?
+      def title?
+        return false unless title
+        @text !~ %r!title=false!i
```

Comment:
This would probably be good to memoize, right?

```ruby
def title?
  return false unless title
  return @is_title if defined?(@is_title)
  @is_title = @test !~ %r!title=false!i
end
```

?